tvb_framework/README_bids_monitor.rst

```diff
@@ -0,0 +1,67 @@
+BIDS data builder and directory monitor
```

Comment:
This file is duplicated, I suggest we keep it only under the bids_monitoring folder. The setup_bids_monitor.py should include it from there.